third_party/proton/test/instrument.py

```diff
@@ -50,10 +50,7 @@ def matmul(a, b, activation=""):
     c = torch.empty((M, N), device=a.device, dtype=a.dtype)
 
     # 1D launch kernel where each block gets its own program.
-    def grid():
-        return (1, )
-
-    matmul_kernel[grid](
+    matmul_kernel[(1, )](
```

Comment:
I guess this test wasn't actually run previously, because the grid function has the wrong signature. It should take a `META` argument.

Reply:
It's not very important at this moment. A new framework and full functionality will be developed in the `proton-dev` branch